code/modules/surgery/prosthetic_replacement.dm

```diff
@@ -53,6 +53,10 @@
 				if(human_target.dna.species.id != bodypart_to_attach.limb_id)
 					organ_rejection_dam = 30
 
+			if(!bodypart_to_attach.can_attach_limb(target))
+				to_chat(user, span_warning("You fail in attaching [target]'s [parse_zone(target_zone)]! Their body has rejected [limb_to_attach]!"))
```

Comment:
```suggestion
				to_chat(user, span_warning("You fail in attaching [target]'s [parse_zone(target_zone)]! Their body has rejected [bodypart_to_attach]!"))
```

yo code ain't compiling playa

Reply:
ur too late

Reply:
nah bruh this was sent 30 seconds before your commit hit remote